staging/src/k8s.io/client-go/rest/client.go

```diff
@@ -37,6 +37,15 @@ const (
 	envBackoffDuration = "KUBE_CLIENT_BACKOFF_DURATION"
 )
 
+const (
+	// default configuration for exponentially backoff delay with jitter
+	// with an initial delay of 250ms, maximum delay of 1m and
+	// a maximum of 10% jitter.
+	defaultBackoffInitialDelay = 1 * time.Second
```

Comment:
I would leave 250ms as you had.
We should just be maxing that with the 1s returned from Retry-After - which you will get if you first address the issue I mentioned below.